docs/guides/prefect-deploy.md

```diff
@@ -466,7 +466,7 @@ Use the tabs below to explore these two deployment creation options.
 
     !!! tip "`prefect.yaml` file flexibility"
         In older versions of Prefect, this file had to be in the root of your repository or project directory and named `prefect.yaml`.
-        Now this file can be located elsewhere and be named differently. To use a custom `prefect.yaml` file, supply the new 
+        Now this file can be located elsewhere and be named differently. You can even have multiple `prefect.yaml` files with the same name in different directories. To use a custom `prefect.yaml` file, supply the new 
```

Comment:
it by default deploys from root dir one named `prefect.yaml` right?

Reply:
Yup, if there's no flag the it defaults to `prefect.yaml` in the dir where `prefect deploy` is getting run (which should be the project's root dir). Are you thinking that's something I should mention?

Reply:
i think so, yeah

Reply:
I mentioned that too!

Reply:
ty!